### ISSUE TITLE
Impersonate - Search user does nothing

### ISSUE THREAD

**Issue by matttheba** [rajyraman/Levelup-for-Dynamics-CRM#280]
Browser Version: [object Navigator]
    Extension Version: 3.6.3
    ----------------------------------------------------------
[    [DESCRIBE ISSUE HERE][](url)](https://rinnainz.sharepoint.com/:v:/s/Dept_IT/ETA6GGGXaeNLkMcGKBhMt4UB-YoD_QkSMUAssoQDsVWagA?e=3PybN4)

**Comment by rajyraman** [comment]
@vcqn11 @vfasil @wikap @Diego-Saavedra-San-Juan itsanwar-dev

Please watch this video https://twitter.com/RajYRaman/status/1634393536018911232?s=20.

This is a text box, but it has autocomplete capabilities. The down arrow to select the user shows only when you hover over with the mouse after hitting search. The issue with the previous version which showed all the users as dropdown broke when there are more than 5000 users, hence this change.

If this gets too confusing, I will split up the Search and User list as separate controls. Please let me know if that is the issue.

**Comment by rajyraman** [comment]
@vfasil @matttheba - How many users approximately in the environment? Is it more than 5000?

**Comment by matttheba** [comment]
No, only 147 and just a third of those are licensed, remainder are past employees

**Comment by rajyraman** [comment]
@vfasil @matttheba - Please try sideloading https://github.com/rajyraman/Levelup-for-Dynamics-CRM/releases/tag/v3.6.4 and see if that fixes it.

**Comment by vfasil** [comment]
Only 226 users.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

________________________________

**Comment by matttheba** [comment]
> @vfasil @matttheba - Please try sideloading https://github.com/rajyraman/Levelup-for-Dynamics-CRM/releases/tag/v3.6.4 and see if that fixes it.

Yes, that works as per your original post

**Comment by vfasil** [comment]
Yes, this version 3.6.4 works. Thanks!

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

**Comment by matttheba** [comment]
I should add though, that the extension icon is not updating to show the username.

**Comment by matttheba** [comment]
Yep works. I did find Search inconsistent though, sometimes I had to click it multiple times.

**Comment by rajyraman** [comment]
@matttheba - Search is querying the Dynamics Web API but via async chrome messages, so there might be a latency on that front. I have submitted the new release to Chrome and Edge. I will look into this issue end of the week and see if anything can be done to improve the experience.

**Comment by rajyraman** [comment]
The new version is live. Please check if that resolves your issue.

https://chrome.google.com/webstore/detail/level-up-for-dynamics-365/bjnkkhimoaclnddigpphpgkfgeggokam?hl=en

**Comment by jason-russell** [comment]
Can you release for Edge too?
I tried installing the chrome version but got an error

![image](https://user-images.githubusercontent.com/14230213/228416043-57405acd-f076-4431-abe3-2722730867f1.png)
![image](https://user-images.githubusercontent.com/14230213/228416086-d3d46297-4952-4703-a80c-11aafa529f03.png)

**Comment by rajyraman** [comment]
@jason-russell - Edge version is now live. But, I did not experience this issue when I installed the extension from Chrome store.

**Comment by vfasil** [comment]
Worked for me. Thanks.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

**Comment by matttheba** [comment]
Search doesn't work for me in Chrome or Edge with the new versions. The little arrow does not show up.

**Comment by rajyraman** [comment]
@matttheba - I am not able to repo this. You can try debugging using source maps.

![image](https://user-images.githubusercontent.com/5035266/229044949-93c6e53c-33a9-42c6-a381-c4a7a26a4176.png)

You can also try checking in Network logs and see if the fetch returned any results.

![image](https://user-images.githubusercontent.com/5035266/229045054-dacbf102-e9dc-4abb-adae-a27cba4603fd.png)

The arrow should show up if there are results and when you hover or click the text box.
